contrib/endpoints/src/api_manager/context/request_context.h

```diff
@@ -163,6 +175,9 @@ class RequestContext {
   // Report().
   std::string auth_authorized_party_;
 
+  // Auth Claims: This is the decoded payload of the JWT token
+  char *auth_claims_;
```

Comment:
We can use std::string to store it.  so you don't need to include grpc headers in order to call gpr_free